### ISSUE TITLE
gaussian_to_xyz minor issue

### ISSUE THREAD

**Issue by songyongshun** [jminuse/chemistry-libs#1]
176 line of filetypes.py
`os.system('/fs/europa/g_pc/vmd-1.9 '+output_file+' > /dev/null')`
Is it necessary besides annoying me?

**Comment by jminuse** [comment]
Yeah, that was specific to the system I used at the time (just a quick way of running VMD for display). You can take the line out, or change the path to your own VMD installation or other molecule viewer.

**Comment by songyongshun** [comment]
I see, thank you!
